Docs/user_guide/modules/resamplescalarvolume.md

```diff
@@ -1,3 +1,9 @@
+:::{warning}
+
+This module is deprecated and will be removed in the future. It is recommended to use the [Crop Volume](cropvolume.md) module.

```

Comment:
"Resample Scalar Volume" is the only module that allows simple setting of output image spacing. "Resample scalar/vector/dwi volume" module requires setting of origin, spacing, axis directions, and extents as well. Crop volume module does not allow setting anisotropic spacing. So, this module is not completely redundant. We could slightly improve "Resample scalar/vector/dwi volume" module (to allow automatic setting of image origin, directions, and extents) and then remove "Resample scalar volume" module completely.

Reply:
It would seem confusing that "Legacy" would not equal "Deprecated and will be removed in the future". 

Since "Resample Scalar Volume" is already in the "Legacy" category in the Slicer application, I suggest to keep the deprecation warning. It will be removed in the future, but as indicated there is additional work needed to make it a good experience (single resample module) before it is removed. Whatever the future may hold, "Resample Scalar Volume" module specifically should not be relied on as existing in the future.
![image](https://user-images.githubusercontent.com/15837524/143903893-142a3e6c-4129-4c8d-8525-69c8991d7b9e.png)

How about the following modification?
```suggestion
This module is deprecated and will be removed in the future. It is recommended to use the [Resample Scalar/Vector/DWI Volume](ResampleScalarVectorDWIVolumeParameters.md) module or the [Crop Volume](cropvolume.md) module.

```